wp-api.js

```diff
@@ -1,24 +1,19 @@
-(function( window, undefined ) {
+'use strict';
 
-	'use strict';
-
-	function WP_API() {
-		this.models = {};
-		this.collections = {};
-		this.views = {};
-	}
+( function( WP_API_Settings, Backbone, _, window, undefined ) {
 
 	window.wp = window.wp || {};
-	wp.api = wp.api || new WP_API();
-
-})( window );
 
-(function( Backbone, _, window, undefined ) {
-
-	//'use strict';
+	wp.api = {
+		models: {},
+		collections: {},
+		utils: {}
+	};
 
-	// ECMAScript 5 shim, from MDN
-	// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString
+	/**
+	 * ECMAScript 5 shim, from MDN
+	 * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString
+	 */
 	if ( ! Date.prototype.toISOString ) {
 		var pad = function( number ) {
```

Comment:
Nitpick, but `var pad` ought to live outside the `if` statement: that declaration will be hoisted anyway so it's generally preferable not to declare variables inside conditional blocks.